test/libsolutil/IpfsHash.cpp

```diff
@@ -60,15 +60,23 @@ BOOST_AUTO_TEST_CASE(test_largest_unchunked)
 	BOOST_CHECK_EQUAL(ipfsHashBase58(data), "QmbNDspMkzkMFKyS3eCJGedG7GWRQHSCzJCZLjxP7wyVAx");
 }
 
-// TODO This needs chunking implemented
-//BOOST_AUTO_TEST_CASE(test_large)
-//{
-//	size_t length = 1310710;
-//	string data;
-//	data.resize(length, 0);
-//	BOOST_REQUIRE_EQUAL(data.size(), length);
-//	BOOST_CHECK_EQUAL(ipfsHashBase58(data), "QmNg7BJo8gEMDK8yGQbHEwPtycesnE6FUULX5iVd5TAL9f");
-//}
+BOOST_AUTO_TEST_CASE(test_large)
+{
+	size_t length = 1310710;
+	string data;
+	data.resize(length, 0);
+	BOOST_REQUIRE_EQUAL(data.size(), length);
+	BOOST_CHECK_EQUAL(ipfsHashBase58(data), "QmNg7BJo8gEMDK8yGQbHEwPtycesnE6FUULX5iVd5TAL9f");
```

Comment:
@mijovic did you check locally that these hashes are indeed correct?

Reply:
Yes, I checked locally, as well as in https://dag.ipfs.io/ with correct settings.